Week-4/session-10/callBindApply.js

```diff
@@ -0,0 +1,38 @@
+class Calculator{
+  add(num1,num2){
+      return num1+num2
+  }
+  subtract(num1,num2){
+      return num1-num2
+  }
+  multiply(num1,num2){
+      return num1*num2
+  }
+  divide(num1,num2){
+      return num1/num2
+  }
+}
+
+class ScientificCalculator extends Calculator {
+  square(num){
+      return num**2
+  }
+  cube(num){
+      return num**3
+  }
+  power(num,raise){
+      return num**raise
+  }
+  multiplyByTwo(num) {
+      return num * 2;
+  }
+  powerOfThree(num){
+      return num**3
+  }
+}
+
+let obj = new ScientificCalculator
```

Comment:
When instantiating ScientificCalculator, use parentheses even if there are no arguments: let obj = new ScientificCalculator(). This is more consistent and clearer, especially for other developers who might read your code